src/CssManager.php

```diff
@@ -2312,13 +2318,13 @@ private function getFileContents($path)
 
 			$contents = @file_get_contents($localpath);
 
-		} elseif (!$contents && !ini_get('allow_url_fopen') && function_exists('curl_init')) { // if not use full URL
+		} elseif ($this->remoteContentFetcher !== null) { // if not use full URL
```

Comment:
RemoteContentFetcher cannot be null when the constructor definition is changed. Is the ommited `!$contents` condition superfluous?

Reply:
Yep, the `!$contents` check is superfluous, as it has already been checked on line 2297.